inspire/base/templates/format/record/Inspire_HTML_detailed_macros.tpl

```diff
@@ -0,0 +1,327 @@
+{#
+# This file is part of INSPIRE.
+# Copyright (C) 2015 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+{% from "format/record/Inspire_Default_HTML_general_macros.tpl" import record_cite_modal, record_abstract with context %}
+
+{% macro record_collection_heading() %}
+{% if record.get('collections') %}
+    {% set collections = record.collections %}
+    {% for collection in collections %}
+      {% set collection_name = collection.get('primary') %}
+      {% if collection_name == "HEP" %}
+        <span id="search-hep">Search Literature &gt;</span>
+      {% elif collection_name == "HEPNAMES" %}
+        <span id="search-hep">Search Authors &gt;</span>
+      {% elif collection_name == "CONFERENCES" %}
+        <span id="search-hep">Search Conferences &gt;</span>
+      {% elif collection_name == "JOB" %}
+        <span id="search-hep">Search Jobs &gt;</span>
+      {% endif %}
+    {% endfor %}
+  {% endif %}
+	<span id="title">{{ record_title() }}</span>
+{% endmacro %}
+
+{% macro record_title() %}
+	{% block title %}
+		{% if record['title'] | is_list %}
```

Comment:
same here, watch out as the data model has changed https://github.com/inspirehep/inspire-next/blob/master/inspire/dojson/hep/schemas/hep-0.0.1.json#L211